src/components/map/MapService.js

```diff
@@ -916,24 +937,22 @@ goog.require('ga_urlutils_service');
                 attributionUrl: 'http://www.swisstopo.admin.ch/internet/' +
                     'swisstopo/en/home/products/height/swissALTI3D.html'
               };
-              // WMTS (not MapProxy)
+              // WMTS (MapProxy)
```

Comment:
It's only comment, but swissimage is pre-generated in 4326, so should not pass over mapproxy.